config.json

```diff
@@ -51,7 +53,8 @@
         },
         {
           "name": "headers",
-          "viewName": "Headers (Obj)",
+          "viewName": "Headers (Object)",
+          "default": {},
```

Comment:
Does that really work and doesn't cause issues? What does that put into the input field?